discodeit/src/main/java/com/sprint/mission/discodeit/config/MDCLoggingInterceptor.java

```diff
@@ -0,0 +1,81 @@
+package com.sprint.mission.discodeit.config;
+
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import lombok.extern.slf4j.Slf4j;
+import org.slf4j.MDC;
+import org.springframework.web.servlet.HandlerInterceptor;
+
+import java.net.Inet6Address;
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+
+/**
+ * MDC(Mapped Diagnostic Context)
+ * 요청마다 MDC에 컨텍스트 정보를 추가하는 인터셉터
+ * Interceptor : URL 기준으로 Spring 레벨에서 필터를 제공해주는 인터페이스
+ */
+@Slf4j
+public class MDCLoggingInterceptor implements HandlerInterceptor {
+
+    // MDC 로깅에 사용되는 상수 정의
+    public static final String REQUEST_ID = "requestId";
+    public static final String REQUEST_METHOD = "requestMethod";
+    public static final String REQUEST_URI = "requestUri";
+    public static final String REQUEST_ID_HEADER = "Discodeit-Request-ID";
+
+    // 요청 이전에 받아서 처리하는 핸들러
+    @Override
+    public boolean preHandle(HttpServletRequest request, HttpServletResponse response, Object handler) throws Exception {
+        // 요청 ID 생성 (IP + UUID 일부)
+        String requestId = getClientIp(request) + "-" + java.util.UUID.randomUUID().toString().substring(0, 8);
```

Comment:
IP를 추출해서 처리하신 이유가 있을까요?
보통 UUID만으로도 충분합니다. 

```suggestion
String requestId = UUID.randomUUID().toString().replaceAll("-", "");

```